oauth.go

```diff
@@ -40,3 +45,33 @@ func (app App) GetAccessToken(shopName string, code string) (string, error) {
 	err = client.Do(req, token)
 	return token.Token, err
 }
+
+// Verify a message against a message HMAC
+func (app App) VerifyMessage(message, messageMAC string) bool {
```

Comment:
What's `app`? Why do you have a type after `messageMAC` and not after `message`?